cooperatives/nhcis

```diff
@@ -0,0 +1,31 @@
+---
+title: New Hampshire Connections Information System
+parent: Cooperatives
+---
+
+# New Hampshire Connections Information System
+
+|                   |                                          |
+|:------------------|:-----------------------------------------|
+| model             | 
+| service type      | Information Technology
+| country           | United States
+| states            | NH
```

Comment:
@abbeyns149 The description for this system says "developed to date in partnership with Massachusetts." Can you add MA to the states list here, and do some research to learn more about the MA side of this software development? It seems the website linked just has info from NH's perspective, so there's probably some more info on a MA state website about how MA has been involved with and uses the software. It might take some digging.